types/evse_manager.yaml

```diff
@@ -553,4 +591,4 @@ types:
           type: object
           $ref: /evse_manager#/Connector
         minItems: 1
-        maxItems: 128
+        maxItems: 128
```

Comment:
No newline at end of file

Reply:
more applicable to #505 where it will need to be addressed.